ui.apps/src/main/content/jcr_root/apps/__appsFolderName__/components/example-component/webpack.module/index.scss

```diff
@@ -1,4 +1,5 @@
-$background: skyblue;
+@import 'scss/variables';
+
 // $background: springgreen;  // <- changes background color of page
```

Comment:
stylelint should give an error for a named color. Use a lowercase 6 character hex color or rgba instead 😛

Reply:
It does give an error if I use your rules, well spotted! :) However, because I'm not loading [stylelint-config-infield](https://github.com/infielddigital/stylelint-config-infield) by default, nobody will be bothered by that warning. If someone adds a stylelint config that checks for that rule, this example at least demonstrates that the config works.